app/uk/gov/hmrc/gform/gform/FormController.scala

```diff
@@ -550,7 +550,9 @@ class FormController(
                           )
                       }
 
-                    updatePostcodeLookup.fold(continueJourney) { case (formComponentId, _) =>
+                    val continueJourneyWithoutRedirect = maybeRedirect.fold(continueJourney)(r => Redirect(r))
```

Comment:
`continueJourneyWithoutRedirect` deserves better name, since you just did the redirect, so the name is misleading